content/accounts.md

```diff
@@ -0,0 +1,228 @@
+
+1. A users collection with a standard schema, accessed through [`Meteor.users`](http://docs.meteor.com/#/full/meteor_users), and the client-side singletons [`Meteor.userId()`](http://docs.meteor.com/#/full/meteor_userid) and [`Meteor.user()`](http://docs.meteor.com/#/full/meteor_user), which represent the login state on the client
+2. A variety of helpful other generic methods to keep track of login state, log out, validate users, etc. Visit the [Accounts section of the docs](http://docs.meteor.com/#/full/accounts_api) to find a complete list.
+3. An API for registering new login handlers, which is used by all of the other accounts packages to integrate with the accounts system. There isn't any official documentation for this API, but you can [read more about it on the MeteorHacks blog](https://meteorhacks.com/extending-meteor-accounts).
+
+Usually, you don't need to include `accounts-base` yourself since it's added for you if you use `accounts-password` or similar, but it's good to be aware of what is what.
+
+## Fast prototyping with `accounts-ui`
+
+Often, a complicated accounts system is not the first thing you want to build when you're starting out with a new app, so it's useful to have something you can just drop in quickly. This is where `accounts-ui` comes in - it's just one line that you drop into your app to get an accounts system. To add it:
+
+```js
+meteor add accounts-ui
+```
+
+Then just include it anywhere in a Blaze template:
+
+```html
+{{> loginButtons}}
+```
+
+Then, make sure to pick a login provider; they will automatically integrate with `accounts-ui`:
+
+```sh
+# pick one or more of the below
+meteor add accounts-password
+meteor add accounts-facebook
+meteor add accounts-google
+meteor add accounts-github
+meteor add accounts-twitter
+meteor add accounts-meetup
+meteor add accounts-meteor-developer
+```
+
+[XXX screenshot of accounts ui in action]
+
+Now just open your app, follow the configuration steps, and you're good to go - if you've done the [Meteor tutorial](https://www.meteor.com/tutorials/blaze/adding-user-accounts), you've already seen this in action. Of course, in a production application, you probably want a more custom user interface and some logic to have a more tailored UX, but that's why we have the rest of this guide.
+
+## Password login
+
+Meteor comes with a secure and fully-featured password login system out of the box. To use it, add the package:
+
+```sh
+meteor add accounts-password
+```
+
+To see what options are available to you, read the complete description of the [`accounts-password` API in the Meteor docs](http://docs.meteor.com/#/full/accounts_passwords).
+
+### Requiring username or email
+
+Be default, the `Accounts.createUser` function provided by `accounts-password` allows you to create an account with a username, email, or both. Most apps expect a specific combination of the two, so you will certainly want to validate the new user creation:
```

Comment:
Do you need to do this? I guess this stops creating a user in some other way on the server?